backend/src/main/java/pl/edu/agh/apdvbackend/models/body_models/auth/UserWithRoles.java

```diff
@@ -0,0 +1,13 @@
+package pl.edu.agh.apdvbackend.models.body_models.auth;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import pl.edu.agh.apdvbackend.models.database.Role;
+
+public record UserWithRoles(
+        @Schema(required = true) Long id,
+        @Schema(required = true) String name,
```

Comment:
name poki co zostaje?

Reply:
Tak, wyrzucę to w kolejnych taskach, żeby nie mieszać 